Engine/EditorUI/Panel/PanelStateMachine.h

```diff
@@ -47,10 +64,16 @@ class PanelStateMachine : public Panel
 	void LeftPanel();
 	void CreateNodeMenu();
 
+	//polishing node editor
+	ImDrawList* draw_list = nullptr;
+	const float NODE_SLOT_RADIUS = 6.2f;
```

Comment:
If you don't need this const in the header you can put in under a namespace in the cpp file. `namespace { const float NODE_SLOT_RADIUS = 6.2f; }`